siphon/catalog.py

```diff
@@ -335,15 +337,14 @@ def make_access_urls(self, catalog_url, all_services, metadata=None):
         self.access_urls = access_urls
 
     def add_access_element_info(self, access_element):
+        """Create an access method from a catalog element."""
```

Comment:
Why is it that sometimes we use a regular string

~~~
"""blah blah blah"""
~~~

but others, we use a raw string (like line 347 below)

~~~
r"""blah blah blah"""
~~~